src/cljs/rems/administration/licenses.cljs

```diff
@@ -66,11 +67,12 @@
 (rf/reg-sub
  ::licenses-table-rows
  (fn [_ _]
-   [(rf/subscribe [::licenses])])
- (fn [[licenses] _]
+   [(rf/subscribe [::licenses])
+    (rf/subscribe [:language])])
+ (fn [[licenses language] _]
    (map (fn [license]
           {:key (:id license)
-           :title {:value (:title license)}
+           :title {:value (get-catalogue-item-title license language)} ; XXX: not really catalogue item, but the structure is the same
```

Comment:
Could this function be renamed to something like `get-localized-title` and then it would be nicer to use in these different places? Ideally we would use the same localization structure with all these different concepts and then we could use the same fns too.

Reply:
Done. For data from V2 APIs (forms and applications) it's possible to use `rems.text/localized` for all localized strings, but catalogue items and licenses still use the old style .